packages/mdc-list/README.md

```diff
@@ -105,103 +108,103 @@ While in theory you can add any number of "lines" to a list item, you can use th
 </ul>
 ```
 
-### List "Detail" Elements
+### List "Tile" Elements
 
-As mentioned in the spec, lists can contain primary and secondary actions. It can also contain
-things such as avatars, icons, interactive controls, etc. We call these items *detail* items. Lists
-can contain 1 **start detail** item and/or 1 **end detail** item that are positioned at the start
-and end of the list, respectively. These items are correctly flipped in RTL contexts.
+As mentioned in the spec, lists can contain primary and secondary actions. They can also contain
+things such as avatars, icons, interactive controls, etc. We call all of these items *tiles*. Lists
+can contain 1 **supporting graphic** tile and/or 1 **meta data** tile that are positioned at the start
+and end of the list, respectively. These tiles are correctly flipped in RTL contexts.
```

Comment:
"of the list" should be "of the list item", right?

Reply:
Done